content/soarian/dstu2/diagnostic/observation.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Observation | Soarian DSTU 2 API 
+---
+ 
+# Observation
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Observation resource query,the API returns observation names and values for elements that the Soarian client has configured as appropriate for the API. The ONC (Office of the National Coordinator) CCDS (common clinical data set) definition includes at least two types of data relevant to this resource: smoking status and vital signs. The API will return any data, not marked as erroneous, the client has configured as relevant for the API. Note that ONC has clarified CCDS sex as the sex recorded at birth and this may be included in the Observation resource if client-configured as part of the social history template. Data is returned in the sequence defined for these sections. The configuration of qualifying data (data configured for social history and vital signs sections) is a single configuration that applies to both the API and the Consolidated Clinical Document Architecture (C-CDA) document, social history and vital signs sections, available to the patients. 
+
+The API handles vital signs similarly to results in the DiagnosticReport resource; the API returns vital sign names, values, and units of measure in reverse chronological order by charted date. Since each vital sign value is individually dated and the value is unambiguously relevant for that date, this resource supports date-based queries for vital signs. Vital signs include:
+ 
+*  Temperature
+*  Systolic BP
+*  Diastolic BP
+*  Respiratory Rate
+*  Pulse (Heart Rate)
+*  SPO2
+*  Height
+*  Weight
+*  Inhaled O2 saturation       
+
+Social History data, such as smoking status, birth sex or any additional client-configured qualifying data, reflects a currently charted state and the most current values will always be returned. ONC requires specific smoking status values and codes which will be returned in the structured portion of the response; charted names will be represented in the narrative extension.
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_observation, :dstu2) %>
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+## Search
+
+	GET /Observation?:parameters
+	
+_Implementation Notes_
+
+* The API returns the most current charted Observation list. 
+
+* The API returns values in reverse chronological order by the most currently charted date.
+
+* The API returns charted content in final status.
+
+* The API returns charted content that is not marked as erroneous.
```

Comment:
Updated the document as 'The API will not return charted content that is marked as cancelled or entered-in-error status' 
In Soarian there is no concept of registered, preliminary, or amended.